_stbt/core.py

```diff
@@ -546,6 +546,42 @@ def _load_template(template):
         return _AnnotatedTemplate(image, relative_filename, absolute_filename)
 
 
+def load_image(filename, flags=cv2.CV_LOAD_IMAGE_COLOR):
+    """Find & read an image from disk.
+
+    If given a relative filename, this will search in the directory of the
+    Python file that called ``load_image``, then in the directory of that
+    file's caller, etc. This allows you to use ``load_image`` in a helper
+    function, and then call that helper function from a different Python file
+    passing in a filename relative to the caller.
+
+    Finally this will search in the current working directory. This allows
+    loading an image that you had previously saved to disk during the same
+    test run.
+
+    This is the same lookup algorithm used by `stbt.match` and similar
+    functions.
+
+    :type filename: str or unicode
+    :param filename: A relative or absolute filename.
+
+    :param flags: Flags to pass to `cv2.imread`.
+
+    :returns: An image in OpenCV format (a `numpy.ndarray` of 8-bit values, 3
+        channel BGR).
+    :raises: `ValueError` if the specified path doesn't exist or isn't a valid
+        image file.
+    """
+
+    absolute_filename = _find_path(filename)
+    if not absolute_filename or not os.path.isfile(absolute_filename):
+        raise ValueError("No such file: %s" % filename)
```

Comment:
Perhaps this should be `IOError` - this is what is usually raised for file not found errors. Arguably `_find_path` should actually be `_open_file_on_path`.  It would avoid races (probably not a problem in practice) and centralise more error handling. You could then use `cv2.imdecode` to get the image out and `file.name` to get the name of the file.